Cargo.toml

```diff
@@ -23,6 +23,7 @@ rustls = "0.22.4"
 webpki-roots = "0.26.6"
 reqwest = "0.12.9"
 thiserror = "2.0.12"
+clap = "4.5"
```

Comment:
you need to also add this dependency to `crates/enclave/Cargo.toml`, where you use it.
```toml
clap.workspace = true
```